internal/execext/exec.go

```diff
@@ -3,6 +3,14 @@ package execext
 import (
 	"context"
 	"errors"
+	"fmt"
+	"github.com/docker/distribution/reference"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/jsonmessage"
+	"github.com/docker/docker/pkg/term"
```

Comment:
All imports from packages outside the standard library should be moved to the second block.